build/config/compiler/BUILD.gn

```diff
@@ -922,6 +923,10 @@ if (is_win) {
       common_optimize_on_ldflags += [ "-Wl,--as-needed" ]
     }
   }
+
+  if (!enable_profiling) {
+    common_optimize_on_cflags += [ "-fvisibility=hidden" ]
```

Comment:
Can we handle this in `//build/config/gcc:symbol_visibility_hidden` instead please?